ui/src/pages/backoffice/Patron/PatronDetails/components/PatronCurrentLoans/PatronCurrentLoans.js

```diff
@@ -32,21 +32,19 @@ export default class PatronCurrentLoans extends Component {
 
   viewDetails = ({ row }) => {
     return (
-      <Button
-        as={Link}
+      <Link
         to={BackOfficeRoutes.loanDetailsFor(row.metadata.pid)}
-        compact
-        icon="info"
         data-test={row.metadata.pid}
-      />
+      >
+        {row.metadata.pid}
+      </Link>
     );
   };
 
   render() {
     const { data, isLoading, error } = this.props;
     const columns = [
-      { title: '', field: '', formatter: this.viewDetails },
-      { title: 'ID', field: 'metadata.pid' },
+      { title: 'PID', field: 'metadata.pid', formatter: this.viewDetails },
```

Comment:
```suggestion
      { title: 'PID', field: '', formatter: this.viewDetails },
```
or even:
```
{ title: 'PID', formatter: this.viewDetails },
```

While refactoring around, I found misleading seeing both because when we have the formatter, the `field` field is totally ignored.

Reply:
ah for me it was a shortcut to see from which field I was taking the data, but you are right it might lead to inconsistencies